pylint/checkers/variables.py

```diff
@@ -1656,10 +1644,10 @@ def _loopvar_name(self, node, name):
 
             # Consider sequences.
             sequences = (
-                astroid.List,
-                astroid.Tuple,
-                astroid.Dict,
-                astroid.Set,
+                nodes.List,
+                nodes.Tuple,
+                nodes.Dict,
+                nodes.Set,
                 astroid.objects.FrozenSet,
```

Comment:
Look like there's other class too that should go in astroid.nodes ?

Reply:
From the `astroid.objects` docstring:
```py
"""
Inference objects are a way to represent composite AST nodes,
which are used only as inference results, so they can't be found in the
original AST tree. [...]
"""
```

As they are only used in inference, I don't think they should move to the `nodes` package. At least not for now.